GetIntoTeachingApi/Jobs/BaseJob.cs

```diff
@@ -6,7 +6,7 @@ namespace GetIntoTeachingApi.Jobs
 {
     public abstract class BaseJob
     {
-        protected readonly IEnv Env;
+        internal readonly IEnv Env;

```

Comment:
Looks good, but the protection level here is worse. Wonder if this could be an auto property along the lines of `protected  IEnv Env { get; init; }`

Reply:
Just noticed that the casing is wrong, wonder if that is what is upsetting the static analyser. I think its recommended to use auto properties where possible anyway.